### PR TITLE
Integration test configuration enabled

### PR DESCRIPTION
**Pull request by dmvieira**

### CODE REVIEW

File: requirements/requirements_base.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-async-worker==0.11.1
\ No newline at end of file
+async-worker==0.11.1
+aioamqp==0.14.0
```
Comment (gligneul) [inline requirements/requirements_base.txt]: This shouldn't be necessary anymore

Comment (dmvieira) [reply requirements/requirements_base.txt]: It is... Just new versions of async-worker are covered

File: tests_integration/test_rabbitmq_integration.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+from barterdude import BarterDude
+
+from asyncworker import RouteTypes
+from asyncworker.connections import AMQPConnection
+
+
+class RabbitMQConsumerTest(TestCase):
+
+    use_default_loop = True
```
Comment (gligneul) [inline tests_integration/test_rabbitmq_integration.py]: Not used

Comment (dmvieira) [reply tests_integration/test_rabbitmq_integration.py]: For sync tests we need this flag

**Pull request merged**